idos-build/Dockerfile.idos-facetec

```diff
@@ -48,6 +48,11 @@ RUN apt-get update && apt-get install -y \
 COPY $FACETEC_CUSTOM_SERVER_DIRECTORY /home/FaceTec_Custom_Server
 COPY $FACETEC_SERVER_CORE_DIRECTORY /home/FaceTec_Server_Core
 COPY ./OCRTemplates /home/OCRTemplates
+
+# FaceTec mongo storage modifications (don't store face images, just empty strings)
+RUN sed -i 's#faceScanRequest.auditTrailImage.getBase64()#""#' /home/FaceTec_Custom_Server/src/main/java/com/facetec/customserver/database/MongoDatabase.java
+RUN sed -i 's#faceScanRequest.lowQualityAuditTrailImage.getBase64()#""#' /home/FaceTec_Custom_Server/src/main/java/com/facetec/customserver/database/MongoDatabase.java
```

Comment:
I'd maybe consider adding checksums (even md5 would be fine) for the file here, just to make sure things can only go as expected.